tracing-jaxrs/build.gradle

```diff
@@ -19,9 +19,13 @@ apply plugin: 'org.inferred.processors'
 apply plugin: 'com.palantir.external-publish-jar'
 apply plugin: 'com.palantir.revapi'
 
+versionsLock {
+    disableJavaPluginDefaults()
+}
+
 dependencies {
     api project(":tracing")
-    api "jakarta.ws.rs:jakarta.ws.rs-api"
+    api "javax.ws.rs:javax.ws.rs-api:2.1.1"
```

Comment:
can we put this in versions.props or do we explicitly have to pin the old javax deps?

Reply:
This was partially stylistic just so it was explicit where the deps were coming from. Let me see if the latest build yells at me for anything (like unused pins).